ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -1577,6 +1577,36 @@ def test_tensorflow_reverse(
     )
 
 
+# reverse_sequence
+@handle_frontend_test(
+    fn_free="tensorflow.reverse_sequence",
+    dtype_and_axis=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        length_dtype=["int64"],
```

Comment:
Getting the following error because of this `length_dtype` variable 
```
ERROR ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py - TypeError: accept.<locals>.dtype_and_values() got an unexpected keyword argument 'length_dtype'
```